elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -37,18 +38,23 @@ async def verify_attached_bugs_cli(runtime: Runtime, verify_bug_status: bool, ad
         exit(1)
     if runtime.use_jira:
         await verify_attached_bugs(runtime, verify_bug_status, advisories, verify_flaws, True)
-    await verify_attached_bugs(runtime, verify_bug_status, advisories, verify_flaws, False)
+    else:
```

Comment:
i think it should query bugzilla always.

Reply:
Yes this part isn't complete yet - use_jira will not be an option anywhere. We will by default be searching everywhere